ledger/metrics/src/test/suite/scala/com/daml/metrics/InstrumentedSourceSpec.scala

```diff
@@ -141,6 +143,83 @@ final class InstrumentedSourceSpec extends AsyncFlatSpec with Matchers with Akka
     }
   }
 
+  behavior of "InstrumentedSource.bufferedSource"
+
+  def throttledTest(producerMaxSpeed: Int, consumerMaxSpeed: Int): Future[List[Long]] = {
+    val counter = SamplingCounter(10)
```

Comment:
Can you use `10.millis` here for readability, and convert to a `Long` in the `SamplingCounter` constructor?